frontend-web/webclient/app/Files/FileBrowse.tsx

```diff
@@ -1382,6 +1382,10 @@ function FileBrowse({opts}: {opts?: ResourceBrowserOpts<UFile> & AdditionalResou
                     )
                 });
 
+                browser.on("searchHidden", () => {
+                    browser.open(lastActiveFilePath, true);
```

Comment:
I don't fully remember the semantics of `searchHidden`.  So I have two questions:

1. Can this trigger without searching (i.e. just clicking twice on the search icon)? Does this trigger the expected behavior?
2. If this page was opened on the search page, then it appears that `lastActiveFilePath` is `""`. What does `browser.open("", true)` do and does it make sense?

Reply:
1. Yes, double clicking will trigger this and use the empty string, so this PR should not be accepted in its current state.
2. What "search page" are you referring to?

Reply:
Search and refresh on that page (with the new URL)

Reply:
Seems to revert to `app/drives` in both cases for me.